src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const longTermDays = 7;
+  const longTermDiscount = 50;
+  const shortTermDays = 3;
+  const shortTermDiscount = 20;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md): Using all caps for constants (e.g., LONG_TERM_DAYS, LONG_TERM_DISCOUNT) clearly indicates that these fixed values serve as input parameters, improving code readability and maintainability